Tests/test_SeqIO_PdbIO.py

```diff
@@ -70,14 +74,17 @@ def test_seqres_read(self):
             chain = SeqIO.read("PDB/1A8O." + extension, parser)
             self.assertEqual(chain.id, "1A8O:A")
             self.assertEqual(chain.annotations["chain"], "A")
-            self.assertEqual(str(chain.seq),
-                             "MDIRQGPKEPFRDYVDRFYKTLRAEQASQEVKNWMTETLLVQNANPD"
-                             "CKTILKALGPGATLEEMMTACQG")
+            self.assertEqual(
+                str(chain.seq),
+                "MDIRQGPKEPFRDYVDRFYKTLRAEQASQEVKNWMTETLLVQNANPD"
+                "CKTILKALGPGATLEEMMTACQG",
+            )
```

Comment:
Can we have the string on one line, or is it just over the 88 line limit?

Reply:
it misses by one :(